### ISSUE TITLE
How to run kubeless UI using docker-compose

### ISSUE THREAD

**Issue by migmartri** [kubeless/kubeless-ui#24]
Hi,

docker-compose is able to display the UI but there are some issues connecting the UI to the k8s cluster via the proxy.

Basically the issue is that the UI's  docker envionment does not have access to proxy-url:proxy-port

```
dev_1  | webpack: Compiled successfully.
dev_1  | Mon, 05 Jun 2017 17:37:12 GMT app:server Error: request to http://localhost:8080/api/v1/pods failed, reason: connect ECONNREFUSED 127.0.0.1:8080
```
What would be the best way to achieve this? From the top of my head, we could tell the UI service to run in network mode `host`

```
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -13,3 +13,4 @@ services:
       - .:/app
     ports:
       - "3000:3000"
+    network_mode: host
```

**Comment by sebgoa** [comment]
@kemcake did you use compose to test the UI. Could you check this and add a section in the readme as well ? thanks

**Comment by kemcake** [comment]
Yes that was my main issue with docker-compose, how to actually communicate with k8s...
Otherwise yes I could update the readme, the only issue in the docker-compose dev environment is hot module replacement not working.
